src/routes/(console)/project-[region]-[project]/+error.svelte

```diff
@@ -1,13 +1,31 @@
-<script>
+<script lang="ts">
     import { page } from '$app/state';
     import { Container } from '$lib/layout';
-    import { Typography } from '@appwrite.io/pink-svelte';
+    import { Card, Typography } from '@appwrite.io/pink-svelte';
+    function getResource(id: string) {        
+        id = id.replace('/(console)/project-[region]-[project]/', '');
+        let parts = id.split('/');
+        const resource = parts[0];
+
+        return resource === 'settings' ? 'project' : resource;
+    }
 </script>
 
 <Container>
-    <Typography.Title size="xl"
-        >{'status' in page.error
-            ? page.error.status || 'Invalid Argument'
-            : 'Invalid Argument'}</Typography.Title>
-    <p class="body-text-2 u-bold u-margin-block-start-4">{page.error.message}</p>
+    {#if page.error.type === 'general_resource_blocked'}
+        {@const resource = getResource(page.route.id)}
+        <Card.Base>
+            <Typography.Title size="s">Your {resource} is paused</Typography.Title>
```

Comment:
wouldn't this become `Your functions is paused` or `Your sites is paused`? Should be `Your functions are paused`, etc. Please handle singular and plural cases. Eg: for `project` above.